src/boot/stylesheets/main.scss

```diff
@@ -44,6 +44,18 @@ body {
     	color: $gray;
     }
 
+    button {
+		background-color: transparent;
```

Comment:
> Is your definition of a "link" when the trigger is just text?

That would be my first thought when scanning a page. Though it can be sometimes be used as an action when we really don't a person to do a particular thing. For example having a button that looks like a button as the primary and a link `cancel` as the secondary action. 

I usually defer to whatever our common design language happens to be, but I was curious on thoughts about this.